### DISCUSSION TITLE
How can I build a settings panel like Fluent Search？

### DISCUSSION THREAD

**Discussion by zjcscut** [AvaloniaUI/Avalonia#10181] [Q&A]
![image](https://user-images.githubusercontent.com/13879735/216501423-08578f93-dc6e-41a1-af92-aedc3792ad22.png)

Is any demo code that could be a reference to create a settings panel like 'Fluent Search' app，or how to refresh the theme and styles immiediately. THX

**Comment by maxkatz6** [comment]
It could be anything like ListBox with custom styles.
Source code of this project isn't available as I see, but you can ask for details here https://github.com/adirh3/Fluent-Search

Also, from my understanding, they use https://github.com/amwx/FluentAvalonia themes and controls.

**Reply by zjcscut** [reply]
@maxkatz6 THX bro, I will try it

**Reply by timunie** [reply]
Let me add, I think @amwx is adding this control into FluentAvalonia. At least I ve seen it recently. So if I were you, I'd check out FluentAvalonia 😉.
